### DISCUSSION TITLE
Schema表单中怎么自定义errors提示？

### DISCUSSION THREAD

**Discussion by Jiazhi-Wong** [alibaba/formily#2129] [Q&A 2.x]
使用@formily/antd组件，使用schema表单，通过pattern来限制输入，我想定义此时的错误提示为”此文案必需包含下载两字“，应该怎么搞？

![image](https://user-images.githubusercontent.com/19643491/132680549-64b948e8-25b3-49d9-9292-f7428b51f74b.png)

**Comment by janryWang** [comment]
请不要问文档已有的问题

**Reply by Jiazhi-Wong** [reply]
翻了一圈文档没找到，大佬能不能贴个链接

**Reply by janryWang** [reply]
https://v2.formilyjs.org/zh-CN/guide/advanced/validate
